### ISSUE TITLE
All doc comment is converted to #[doc = " ..."]

### ISSUE THREAD

**Issue by hatoo** [hatoo/cargo-snippet#13]
```rust
#[snippet]
/// Some comment
fn foo() {}
```

run `$cargo snippet`

```
snippet foo
    #[doc = " Some comment"]
    fn foo() {}

```

Since almost always, we don't use `#[doc = " ..."]`.
So it's justifiable that emit doc comment as `///` instead `#[doc]` always.

 I feel it's good even if this is implemented by text processing.

**Comment by magurotuna** [comment]
I'm trying this issue. 💪

**Comment by hatoo** [comment]
Merged #14
